app/controllers/concerns/enlistment_filters.rb

```diff
@@ -17,6 +17,20 @@ module EnlistmentFilters
     before_action :project_has_code_location?, only: :create
   end
 
+  def create; end
+
+  def new; end
+
+  def index; end
+
+  def show; end
+
+  def edit; end
+
+  def update; end
+
+  def destroy; end
+
```

Comment:
Why does Rubocop require empty methods in a Concern? This is weird and just looks wrong to have this empty, dead code.